src/app/components/app-popup-host/app-popup-host.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, Input, TemplateRef, ViewChild, ViewContainerRef} from '@angular/core';
+
+@Component({
+    selector: 'app-popup-host',
+    templateUrl: './app-popup-host.component.html',
+    styleUrls: ['./app-popup-host.component.css'],
+})
+export class AppPopupHostComponent {
+    @Input()
+    set template(template: TemplateRef<ViewContainerRef> | null) {
+        this.updateTemplate(template);
+    }
+
+    isTemplate: boolean | undefined;
```

Comment:
Рекомендовал бы просто присвоить initial value
```suggestion
    isTemplate = false;
```

Reply:
Согласен
Исправил